### ISSUE TITLE
Some of the changes that made into master a triggering clippy and are blocking other changes on unrelated crates

### ISSUE THREAD

**Issue by maxzaver** [near/nearcore#68]
As I was committing some of the changes to txflow crate, my commit was blocked by the following clippy find in a crate that I synced from the master https://www.dropbox.com/s/zjbr8vrww5bukhf/Screenshot%202018-11-25%2022.02.01.png?dl=0
It is weird that clippy did not block this code when it was initially merged into master.
